docs/guides/universal-profile/check-if-address-is-universal-profile.md

```diff
@@ -0,0 +1,118 @@
+---
+sidebar_label: 'Check if an address is a UP'
+---
+
+# How to check if an address is a Universal Profile?
+
+:::info
+
+You may want to read the [Standard Detection](../../standards/standard-detection.md) page first.
+
+:::
+
+To check if an address is a [Universal Profile](../../standards/universal-profile/introduction.md) we need to perform 3 checks:
+
+## 1) Check address format
+
+This first basic test can be performed via regular expression or 3rd party library function. For example this is how we can achieve this using Web3js [`isAddress`](https://web3js.readthedocs.io/en/v1.2.11/web3-utils.html#isaddress):
+
+```javascript
+import { isAddress } from 'web3-utils';
+
+if (!isAddress(address)) {
+  throw new Error('Invalid address');
+}
+```
+
+## 2) Check EIP-165 `LSP0ERC725Account` interface
+
+This is next check that makes sure we deal with a smart contract that supports the `LSP0ERC725Account` interface ([EIP-165](https://eips.ethereum.org/EIPS/eip-165)). For this we need to set `Contract` instance and call `supportsInterface` method.
+
+<details>
+<summary>EIP-165 supportsInterface ABI</summary>
+
+```javascript
+const eip165ABI = [
+  {
+    type: 'function',
+    stateMutability: 'view',
+    outputs: [
+      {
+        type: 'bool',
+        name: '',
+        internalType: 'bool',
+      },
+    ],
+    name: 'supportsInterface',
+    inputs: [
+      {
+        type: 'bytes4',
+        name: 'interfaceId',
+        internalType: 'bytes4',
+      },
+    ],
+  },
+];
+```
+
+</details>
+
+```javascript
+const eip165Contract = new web3.Contract(eip165ABI, address);
+
+const interfaceId = '0xeb6be62e'; // Can be found here: https://docs.lukso.tech/standards/smart-contracts/interface-ids
```

Comment:
Since we have an import of `'@lukso/lsp-smart-contracts/constants'` , is there a reason why we hardcode the interfaceId of ERC725Account? If the InterfaceId change (which will happen in next release) we need to change that manually, so i would say let's change it to constant imported.

Reply:
Simply didn't have time to make it perfect - i'll add this

Reply:
Added